packages/rum-core/test/formatValidation.ts

```diff
@@ -12,7 +12,7 @@ export function validateFormat(rumEvent: Context) {
   const instance = new ajv({
     allErrors: true,
   })
-  instance
+  void instance
```

Comment:
The `.validate()` returns a `boolean | Promise`. In our case, since we don't have `$async` on our schema, it will return a `boolean` according to [the ajv documentation](https://github.com/ajv-validator/ajv/blob/HEAD/docs/api.md#ajvvalidateschemaorref-object--string-data-any-boolean).
But ESLint still assumes it may return a Promise, hence the `void` operator to make it ignore this.
WDYT?

Reply:
lgtm